idb/analysis.py

```diff
@@ -894,7 +894,10 @@ def get_chunks(self):
 
     def get_stack_change_points(self):
         # ref: ida.wll@0x100793d0
-        v = self.netnode.supval(tag='S', index=0x1000)
+        try:
+            v = self.netnode.supval(tag='S', index=0x1000)
+        except KeyError:
+            return
```

Comment:
i knew we'd have to transition towards this, thanks for helping out!

Reply:
Since IDA's GetFlags return 0 in this case, maybe I should also return 0 here?

Reply:
yeah, we should try to be like IDA's API as close as possible. i was not doing a good job at this.